src/lib/connectors/repos/billing-transactions.js

```diff
@@ -127,6 +127,16 @@ const countByBatchId = async billingBatchId => {
   return parseInt(count);
 };
 
+/**
+ * Updates the isCredited flag for the batch for all transactions
+ * that was credited back
+ * @param {String} billingInvoiceId
+ * @return {Promise}
+ */
+const updateIsCredited = regionId => bookshelf
```

Comment:
You are passing `regionId` but I think you meant `billingInvoiceId`

Reply:
This is correctly region id. when the bill run is sent we want to flag the transactions that has been credited back in invoices that was just sent for the region.

Reply:
Thanks JS doc has now been updated to region Id